src/components/ProcessDocumentScreen/ProcessDocumentScreen.tsx

```diff
@@ -122,96 +101,56 @@ export const ProcessDocumentScreen: FunctionComponent<ProcessDocumentScreen> = (
-                            extension: doc.extension,
-                          })}
-                        </div>
-                        <div className="text-xs text-cloud-500 ml-1">({size})</div>
-                      </div>
+                      <ProcessedDocumentTag
+                        isError
+                        doc={doc}
+                        key={index}
+                        isPending={false}
+                        type={type}
+                        fileName={doc.fileName}
+                      />
                     );
                   })}
                 </div>
-                <div className="flex py-4">
-                  <div className="col-auto ml-auto">
-                    <Button
-                      className="bg-white text-cerulean hover:bg-cloud-100 mb-4"
-                      data-testid="download-fail-button"
-                      onClick={() => {
-                        generateZipFile(failedProcessedDocuments[0].documents);
-                      }}
-                    >
-                      <div className="flex">
-                        <Download />
-                        <div className="text-cerulean ml-2">Download Failed Documents</div>
-                      </div>
-                    </Button>
-                  </div>
-                </div>
-              </div>
+              </>
             )}
             {queueState === QueueState.ERROR && (
-              <div className="bg-red-100 px-8 py-6 rounded-xl">
-                <div className="mb-8">
-                  <div className="text-rose font-bold mb-8" data-testid="error-title">
-                    {`The document(s) could not be ${isIssuingFlow ? "published" : "revoked"} at this time.`}
-                  </div>
-                  <div>
-                    Please contact TradeTrust via email or client representative to resolve your issue. Alternatively,
-                    please try again.
-                  </div>
-                </div>
-                <Button className="bg-white hover:bg-cloud-100 flex mx-auto">
-                  <a
-                    download={generateFileName({
-                      network: config?.network,
-                      fileName: "error-log",
-                      extension: "txt",
-                      hasTimestamp: true,
-                    })}
-                    href={`data:text/plain;charset=UTF-8,${JSON.stringify(error, null, 2)}`}
-                  >
-                    <div className="flex">
-                      <Download />
-                      <div className="text-cerulean ml-2">Download Error Log</div>
-                    </div>
+              <ErrorCard
+                title={`The document(s) could not be ${isIssuingFlow ? "issued" : "revoked"} at this time.`}
+                description={
+                  "Please contact TradeTrust via email or client representative to resolve your issue. Alternatively, please try again."
+                }
```

Comment:
```suggestion
                description={
                  `Please contact TradeTrust via email or client representative to resolve your issue. Alternatively, please try again.`
                }
```

or don't even need `{}` i guess